spec/v3/services/build/find_spec.rb

```diff
@@ -369,4 +369,30 @@
       "resource_type" => "build"
     })}
   end
+
+  describe 'including log_complete' do
+    before do
+      jobs.each do |j|
+        stub_request(:get, "http://travis-logs-notset.example.com:1234/logs/#{j.id}?by=job_id&source=api").
+           with(  headers: {
+            'Accept'=>'*/*',
+            'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+            'Authorization'=>'token notset',
+            'Connection'=>'keep-alive',
+            'Keep-Alive'=>'30',
+            'User-Agent'=>'Faraday v0.15.3'
+             }).
+           to_return(status: 200, body: "{}", headers: {})
+      end
+    end
+
+    before { get("/v3/build/#{build.id}?include=build.log_complete") }
+
+    example { expect(last_response).to be_ok }
+    example do
+      expect(body['log_complete']).to include(
+        'log_complete'
```

Comment:
Does this pass? It looks like it should be `true` or `false`.

Reply:
it passes - all the test does is look to see that a 'log_complete' attribute is included in the payload - it doesn't check to see whether it's true or false. I wasn't sure this was needed and would instead be something tested in a remote_log spec...?

Reply:
Okay, maybe let's look at it on Monday together because it's a bit confusing to me!